src/main/scala/uk/gov/hmrc/SbtSassCompiler.scala

```diff
@@ -55,6 +55,25 @@ object SbtSassCompiler extends AutoPlugin {
           )
         }
 
+        // this would recompile all sass anytime one sass file changed
+        import sbt.util.CacheImplicits._
+        val s                       = streams.value
+        val allSassFiles            = // this would be sourceDir I just didn't want to modify anything else in file
+          (Assets / sourceDirectory).value.globRecursive((Assets / compileSass / includeFilter).value).get
+        val cachedCompiledSassFiles =
+          Tracked.inputChanged[Seq[ModifiedFileInfo], Seq[File]](s.cacheStoreFactory.make("input")) {
```

Comment:
https://www.scala-sbt.org/1.x/docs/Caching.html#FileInfo 

will be list of filename and timestamp `FileModified(/tmp/cache/last,1565855326328)`

not sure if it needs to be sorted, if globRecursive can return in different order each time run